WPFHospitalEditor/Repository/MapObjectRepository.cs

```diff
@@ -40,30 +41,15 @@ public void editAllMapObjectAttributes(MapObject mapObj, MapObject mapObjForUpda
         {
             mapObj.Description = mapObjForUpdate.Description;
             mapObj.Name = mapObjForUpdate.Name;
-            mapObj.selected = mapObjForUpdate.selected;
         }
         public List<MapObject> getAllMapObjects()
         {
             List<MapObject> allMapObjectsToReturn = new List<MapObject>();
             var allMapObjects = getAll();
-            if(SearchResultDialog.selectedObject == null)
+            foreach (MapObject mapObject in allMapObjects)
```

Comment:
GetAllMapObject does everything that GetAll function does

Reply:
GetAll is deleted because GetAllMapObject is used more.